source/index.txt

```diff
@@ -28,53 +22,91 @@
+.. :ref:`scala-databases-collections` section.
 
-- :ref:`scala-bson`
+What's New
+----------
 
-- :ref:`scala-builders`
+For a list of new features and changes in each version, see the :ref:`What's New <scala-whats-new>`
+section.
 
-- :ref:`Validate Driver Artifact Signatures <scala-validate-signatures>`
+.. TODO
+.. Write Data to MongoDB
+.. ---------------------
 
-- `Driver Source GitHub Repository <https://github.com/mongodb/mongo-java-driver>`__
+.. Learn how you can write data to MongoDB in the :ref:`scala-write` section.
 
-- `API Documentation <{+api+}/index.html>`__
+.. TODO
+.. Read Data from MongoDB
+.. ----------------------
 
-- `Previous Versions Documentation <https://mongodb.github.io/mongo-java-driver/>`__
+.. Learn how you can retrieve data from MongoDB in the :ref:`scala-read` section.
 
-To access MongoDB by using Java instead of Scala, use the `Java Sync
-Driver <https://www.mongodb.com/docs/drivers/java/sync/current/>`__ or
-the asynchronous `Java Reactive Streams Driver
-<https://www.mongodb.com/docs/languages/java/reactive-streams-driver/current/>`__.
+.. TODO
+.. Optimize Queries by Using Indexes
+.. ---------------------------------
 
-Compatibility
--------------
+.. Learn how to work with common types of indexes in the :ref:`scala-indexes`
+.. section.
 
-Learn about compatibility between the {+driver-short+} and MongoDB, and
-the driver and Scala in the :ref:`scala-compatibility-tables` section.
+.. TODO
+.. Transform Your Data with Aggregation
+.. ------------------------------------
 
-How To Get Help
----------------
+.. Learn how to use the {+driver-short+} to perform aggregation operations in the
+.. :ref:`scala-aggregation` section.
 
-- Ask questions on our :community-forum:`MongoDB Community Forums <>`
-- Visit our :technical-support:`Support Channels </>`
-- See :driver:`Issues & Help </java/sync/current/issues-and-help/>`
+.. TODO
+.. Configure Operations on Replica Sets
+.. ------------------------------------
 
-Validate Driver Artifact Signatures
------------------------------------
+.. Learn how to configure read and write operations on a replica set in the
+.. :ref:`scala-read-write-config` section.
 
-Learn about how to validate signatures of {+driver-short+} artifacts
-published on Maven in the :ref:`Validate Driver Artifact Signatures <scala-validate-signatures>` section.
\ No newline at end of file
+.. TODO
+.. Issues & Help
+.. -------------
+
+.. Learn how to report bugs, contribute to the driver, and find more resources for
+.. asking questions and receiving help in the :ref:`Issues & Help <scala-issues-and-help>` section.
+
+.. TODO
+.. Compatibility
+.. ------------
+
+.. To learn about the versions of the {+mdb-server+} and the C language that are compatible with 
+.. each version of the {+driver-short+}, see the :ref:`Compatibility <scala-compatibility>` section.
+
+Learn
+------
+
+Visit the Developer Hub to learn more about the {+driver-long+}.
```

Comment:
Are we anticipating adding more subheadings under this learn section? If not we can probably just stick with the Dev Hub section nested under this

Reply:
I don't think there's any scoped additions to this learn section. Do you mean removing the top-level Learn section and pulling the Developer Hub section up to the top level of the page's ToC?

Reply:
Yeah that's what I was thinking, but happy to differ that decision to later when this part is worked on